reporting/api/src/main/java/org/jboss/windup/reporting/model/InlineHintModel.java

```diff
@@ -16,13 +16,14 @@
 @TypeValue(InlineHintModel.TYPE)
 public interface InlineHintModel extends FileLocationModel
 {
-    public static final String TYPE = "Inlinehintmodel";
-    public static final String TITLE = "windupreporting:title";
-    public static final String HINT = "windupreporting:hint";
-    public static final String RULE_ID = "windupreporting:ruleID";
-    public static final String EFFORT = "windupreporting:effort";
-    public static final String LINKS = "windupreporting:links";
-    public static final String FILE_LOCATION_REFERENCE = "windupreporting:fileLocationReference";
+    static final String TYPE = "InlineHintModel:";
```

Comment:
We could, although that seems very verbose to me. It would make gremlin
pretty clumsy, imo.
On Feb 24, 2015 5:29 PM, "Lincoln Baxter, III" <notifications@github.com>
wrote:

> In
> reporting/api/src/main/java/org/jboss/windup/reporting/model/InlineHintModel.java
> <https://github.com/windup/windup/pull/481#discussion_r25299986>:
>
> > @@ -16,13 +16,14 @@
> >  @TypeValue(InlineHintModel.TYPE)
> >  public interface InlineHintModel extends FileLocationModel
> >  {
> > -    public static final String TYPE = "Inlinehintmodel";
> > -    public static final String TITLE = "windupreporting:title";
> > -    public static final String HINT = "windupreporting:hint";
> > -    public static final String RULE_ID = "windupreporting:ruleID";
> > -    public static final String EFFORT = "windupreporting:effort";
> > -    public static final String LINKS = "windupreporting:links";
> > -    public static final String FILE_LOCATION_REFERENCE = "windupreporting:fileLocationReference";
> > +    static final String TYPE = "InlineHintModel:";
>
> We could use the FQCN for this. WDYT? InlineHintModel.class.getName() ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/windup/windup/pull/481/files#r25299986>.
>